src/org/usfirst/frc/team2706/robot/commands/teleop/GearHandlerToggle.java

```diff
@@ -0,0 +1,19 @@
+package org.usfirst.frc.team2706.robot.commands.teleop;
+
+import org.usfirst.frc.team2706.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class GearHandlerToggle extends Command {
```

Comment:
Im really not a fan of there being four different commands that are really small that serve nearly the same purpose, I strongly suggest putting these into the Gear subsystem and then calling them all from one command